cmd/bucket-metadata.go

```diff
@@ -103,6 +104,15 @@ type BucketMetadata struct {
 	bucketTargetConfigMeta map[string]string
 }
 
+// This will generate a new Metadata structure for PanFS gateway mode. However applying such a structure into existing codebase
+// a lot of code should be updated. At the moment every bucket has additional metadate field - PanfsPath. For buckets created
+// in PanFS Gateway mode panfsPath will be set (if user provides such option). For all others buckets - this will be empty string.
+// go:generate msgp -file $GOFILE
+// type PanFsBucketMetadata struct {
```

Comment:
So far you have been using the "Panfs...." spelling, not "PanFs". Let's stick to one version if that's OK.